src/features/pages/partners/deriv-prime/browse-our-faq/deriv-prime-faq.tsx

```diff
@@ -0,0 +1,90 @@
+                align="center"
+                font_family="UBUNTU"
+                weight="normal"
+                size="xlarge"
+            >
+                <Localize translate_text={subsection} />
+            </Typography.Paragraph>
+            <Accordion.Root type="multiple">
+                {faqs.map((faqItem) => (
+                    <Accordion.ShadowItem
+                        item_title={faqItem.question}
+                        key={faqItem.id}
+                        value={faqItem.question}
+                        icon_type="plus"
+                        dir={get_lang_direction()}
+                    >
+                        <Flex.Box
+                            padding_inline="3x"
+                            padding_block="8x"
+                            direction="col"
+                            gap="4x"
+                            dir={get_lang_direction()}
+                            md={{ padding_inline: '7x', padding_block: '12x' }}
+                        >
+                            {faqItem.answers.map((answerItem) => (
+                                <Flex.Box key={answerItem.id}>
+                                    {answerItem.type === 'text' ? (
+                                        <Typography.Paragraph>
+                                            <Localize translate_text={answerItem.text} />
+                                        </Typography.Paragraph>
+                                    ) : answerItem.type === 'list' ? (
+                                        <ul>
+                                            {answerItem.list_title && (
+                                                <Typography.Paragraph>
+                                                    <Localize
+                                                        translate_text={answerItem.list_title}
+                                                    />
+                                                </Typography.Paragraph>
+                                            )}
+                                            {answerItem.list_items &&
+                                                answerItem.list_items.map((answerListItem) => (
+                                                    <li key={answerListItem}>
+                                                        <Typography.Paragraph>
+                                                            <Localize
+                                                                translate_text={answerListItem}
+                                                            />
+                                                        </Typography.Paragraph>
+                                                    </li>
+                                                ))}
+                                        </ul>
+                                    ) : answerItem.type === 'custom_component' ? (
+                                        <div>{answerItem.component}</div>
```

Comment:
`div` is needed here?

Reply:
Yes, for the alignment with the title but could use Flex.Box instead